docs/articles/documentation/test-api/README.md

```diff
@@ -6,7 +6,7 @@ checked: true
 ---
 # Test API
 
-TestCafe allows to write tests using both the TypeScript and modern JavaScript with the latest features like async/await.
+TestCafe allows to write tests using both TypeScript and modern JavaScript with the latest features like async/await.
```

Comment:
TestCafe allows **you** to write tests using both TypeScript and modern JavaScript with the latest features like async/await.